### ISSUE TITLE
PyAutoFEP is it possible to calculate the membrane system at present?

### ISSUE THREAD

**Issue by Teng-Dan** [luancarvalhomartins/PyAutoFEP#27]
Hi, it is mentioned in the manual ‘It automates the building of a simple receptor-ligand system and will fail for complex cases (eg: membrane systems, systems with vacuum regions, solvent mixtures, and systems with nucleic acids on cofactors).’ So I want to know whether A can be applied to the calculation of membrane system. Looking forward to your reply, thanks!

**Comment by luancarvalhomartins** [comment]
Yes, PyAutoFEP is able to handle membrane systems. The automated system builder, however, cannot. Therefore, you will have to prepare the system and supply it as a pre-solvated structure, ideally, alongside the topology. You will also have to use custom mdp files matching your membrane (semi-isotropic pressure coupling? anisotropic? compressibility?). I am planning to add a tutorial for membrane proteins anytime soon. In case you need further help, don't hesitate to contact me here or via email.
